src/controllers/sagesseController.ts

```diff
@@ -0,0 +1,234 @@
+const Request = require('request')
+const jsdom = require('jsdom')
+const { JSDOM } = jsdom
+
+
+export const getSubjectDetails = async (subjectId: string, year: string, res: any) => {
```

Comment:
for res : any you should use the Response type of the module 'express'
import {Response} from 'express;